db/migrations/000010_add_fields_to_environments.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE environments
+ADD COLUMN destroyed_at timestamp WITH TIME ZONE,
+ADD COLUMN is_permanent boolean,
+ADD COLUMN requester text,
+ADD COLUMN namespace text,
```

Comment:
Hmmm, are we going to abandon the existing namespace convention? (Every Terra environment is deployed to a namespace `terra-ENVNAME`)

Reply:
I can remove that, thats probably an artifact from before we nailed down naming. I think we left off w/ terra-Environment.name

Reply:
gotcha, tx!